frame/support/src/traits/tokens/fungible/conformance_tests/inspect_mutate.rs

```diff
@@ -0,0 +1,734 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2017-2023 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::traits::{
+	fungible::{Inspect, Mutate},
+	tokens::{
+		DepositConsequence, Fortitude, Precision, Preservation, Provenance, WithdrawConsequence,
+	},
+};
+use core::fmt::Debug;
+use sp_arithmetic::traits::AtLeast8BitUnsigned;
+
+//
+// mint_into
+//
+
+pub fn mint_into_success<T, AccountId, Balance>(_dust_trap: Option<AccountId>)
```

Comment:
FYI, I think soon we should enforce 

```
#![deny(missing_docs)]
#![deny(clippy::missing_docs_in_private_items)]
```

So you might want to think about it already.